cpg_workflows/large_cohort/ancestry_pca.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 from cpg_utils import Path, to_path
 from cpg_utils.config import get_config
-from gnomad.sample_qc.ancestry import run_pca_with_relateds, assign_population_pcs
+from gnomad_methods.gnomad.sample_qc.ancestry import run_pca_with_relateds, assign_population_pcs
```

Comment:
We should revert this change, since we found out the real issue was that we didn't run pip install on `gnomad_methds`

Reply:
Hmmm after you posted those instructions I did a fresh fork so I'm not sure why they're still there. Will investigate